helper/include/odp/helper/debug.h

```diff
@@ -70,10 +78,8 @@ do { \
 } while (0)
 
 /**
- * Assert macro for applications and helper code
- *
- * No code is generated when ODPH_DEBUG=0. Prints error message and aborts when
- * ODPH_DEBUG=1 and 'cond' is false.
+ * Runtime assertion macro. No code is generated when ODPH_DEBUG=0. Prints error
```

Comment:
Could mention `--enable-helper-debug` configure option for setting `ODPH_DEBUG`.